docs/contributing.rst

```diff
@@ -200,6 +200,33 @@ put up for review.
 
 .. _Transifex: https://www.transifex.com/readthedocs/sphinx-rtd-theme
 
+Versioning
+==========
+
+..
+    TODO make these instructions organization wide. We've talked about
+    standardizing packaging version and tooling here (bumpver instead of
+    bump2version).
```

Comment:
I'm not sure if I would make this organization-wide. It can vary for every project. Release processes also lean towards automation, although I'm not a big fan.

For this project, we could work more with dynamic generation of artifacts for each release.

A versioning guide should contain more details on the semantics of the versions. For instance, can a patch release contain translations? What can be changed in a minor release? How do we handle deprecation? Etc.